photutils/background/core.py

```diff
@@ -32,6 +32,38 @@
            'MADStdBackgroundRMS', 'BiweightMidvarianceBackgroundRMS']
 
 
+def _masked_median(data, axis=None):
+    """
+    Calculate the median of a (masked) array.
+
+    This function is necessary for a consistent interface across all
+    numpy versions.  An bug was introduced in numpy v1.10 where
+    `numpy.ma.median` (with ``axis=None``) returns a single-valued
```

Comment:
No, I wanted to distinguish it as np.ma.median, otherwise it would render simply as "median" (which would be likely interpreted as np.median).  Also, this is a private function, so the the docs will never be shown. :smile: